<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarTable</class>
 <widget class="QWidget" name="CarTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>691</width>
     <height>341</height>
    </rect>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::IgnoreAction</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideNone</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSearch">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>50</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonAdd">
       <property name="text">
        <string>新增</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMod">
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonInsure">
       <property name="text">
        <string>增加保险记录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMaintain">
       <property name="text">
        <string>增加保养记录</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonShowAll">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>显示全部</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入车牌号进行查询</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButtonSearch</tabstop>
  <tabstop>pushButtonShowAll</tabstop>
  <tabstop>pushButtonAdd</tabstop>
  <tabstop>pushButtonMod</tabstop>
  <tabstop>pushButtonInsure</tabstop>
  <tabstop>pushButtonMaintain</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
